aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/customize/s3/S3Decorator.kt

```diff
@@ -56,6 +56,7 @@ class S3Decorator : ClientCodegenDecorator {
             ShapeId.from("com.amazonaws.s3#CreateSessionOutput"),
             // API returns GetObjectAttributes_Response_ instead of Output
             ShapeId.from("com.amazonaws.s3#GetObjectAttributesOutput"),
+            ShapeId.from("com.amazonaws.s3#ListDirectoryBucketsOutput"),
```

Comment:
Add a comment for this one explaining why, like the others above it.

Reply:
Added in b29da13